cli/src/commands/branch/rename.rs

```diff
@@ -60,11 +60,11 @@ pub fn cmd_branch_rename(
     if has_tracked_remote_branches(view, old_branch) {
         writeln!(
             ui.warning_default(),
-            "Branch {old_branch} has tracking remote branches which were not renamed."
+            "Branch {old_branch} has tracked remote branches which were not renamed."
```

Comment:
(Optional): I would prefer "is tracking remote branches" to "has tracked", since the latter sounds like past tense.

Another option (my second choice) would be "Tracking remote branches for branch {old_branch} were not renamed".

Thanks for fixing this, BTW!

Reply:
Good point. I choose the latter. The former could be parsed as "[Branch] is [tracking remote branches]".

Reply:
Ah, thanks for finding and fixing the typo in my suggestion. What you merged is correct, I think.